bin/update-metadata-permissions.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
```

Comment:
Recommend adding a doc string that describes what this script is for. Particularly what the state of metadata was before and what it is after running, so down the road, we know whether we want to run this or maybe even delete it when everything's migrated.